hi-ml-azure/src/health_azure/paths.py

```diff
@@ -44,4 +44,5 @@ def git_repo_root_folder() -> Path:
     """
     if not is_himl_used_from_git_repo():
         raise ValueError("This function can only be used if the HI-ML package is used directly from the git repo.")
-    return Path(__file__).parent.parent.parent.parent
+    current_file = Path(__file__).resolve()
```

Comment:
Think the docstring needs to be updated here - if hi-ml has been installed as a pip package this will raise an error, rather than return None